lib/frp/src/core/nodes/dynamic.rs

```diff
@@ -131,3 +217,104 @@ impl<Out:Value> From<&Dynamic<Out>> for Behavior<Out> {
         t.behavior.clone_ref()
     }
 }
+
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::frp_def;
+    use std::panic::catch_unwind;
```

Comment:
I see here many warnings marked by Github Actions, that the `use std::panic::catch_unwind;` is an unused import. And oddly the CI linter does not complain about anything (the all checks passed)